code/controllers/subsystem/processing/orbits.dm

```diff
@@ -184,6 +184,10 @@ PROCESSING_SUBSYSTEM_DEF(orbits)
 		for(var/M in computer.viewing_mobs)
 			computer.update_static_data(M)
 
+/datum/controller/subsystem/processing/orbits/proc/get_orbital_map_name_from_z(my_z)
+	var/datum/orbital_object/O = assoc_z_levels["[my_z]"]
+	return O.name
```

Comment:
There should be documentation that confirms this is a virtual z-level.
This also needs handling for cases where O is null (return null?)